### PR TITLE
Update base-ind.yaml

### PR DESCRIPTION
**Pull request by Norbert635**
Translated some things and fixed some

### CODE REVIEW

File: translations/base-ind.yaml [hunk 1]
```diff
@@ -4,17 +4,17 @@ steamPage:
         yang meluas tanpa batas.
     discordLinkShort: Official Discord
     intro: >-
-        Shapez.io is a relaxed game in which you have to build factories for the
-        automated production of geometric shapes.
+        Shapez.io adalah game santai dimana anda harus membuat pabrik untuk
```
Comment (KaiSforza) [inline translations/base-ind.yaml:7]: Trailing spaces

File: translations/base-ind.yaml [hunk 2]
```diff
@@ -489,123 +489,121 @@ buildings:
...
-                shape, color or boolean "1")
+            name: Gerbang NOT
+            description: Mengeluarkan  boolean "1" jika input adalah tidak benar. (Benar berarti sebuah bentuk,
+                warna atau boolean "1")
         xor:
-            name: XOR Gate
-            description: Emits a boolean "1" if one of the inputs is truthy, but not both.
-                (Truthy means shape, color or boolean "1")
+            name: Gerbang XOR
+            description: , namun bukan keduanya.
```
Comment (KaiSforza) [inline translations/base-ind.yaml:534]: I don't speak malay, but it feels like something is missing.

It's also causing an error in the file, as well.

File: translations/base-ind.yaml [hunk 3]
```diff
@@ -489,123 +489,121 @@ buildings:
...
+                warna atau boolean "1")
     filter:
         default:
             name: Filter
-            description: Connect a signal to route all matching items to the top and the
-                remaining to the right. Can be controlled with boolean signals
-                too.
+            description: Hubungkan sebuah sinyal untuk merutekan semua benda yang cocok ke atas dan
+                sisanya ke kanan. Dapat juga dikontrol dengan sinyal boolean
+
```
Comment (KaiSforza) [inline translations/base-ind.yaml:554]: It's pointing at trailing spaces here, but I don't see them?

File: translations/base-ind.yaml [hunk 4]
```diff
@@ -489,123 +489,121 @@ buildings:
...
+            name: Layar
+            description: Hubungkan dengan sebuah sinyal untuk ditunjukkan pada layar - Dapat berupa bentuk,
+                warna atau boolean.
     reader:
         default:
-            name: Belt Reader
-            description: Allows to measure the average belt throughput. Outputs the last
-                read item on the wires layer (once unlocked).
+            name: Pembaca Sabuk Konveyor
+            description: Memungkinkan untuk mengukur rata-rata benda yang melewati sabuk konveyor. Mengeluarkan output benda terakhir
```
Comment (KaiSforza) [inline translations/base-ind.yaml:563]: More trailing spaces at the end of the line.

File: translations/base-ind.yaml [hunk 5]
```diff
@@ -489,123 +489,121 @@ buildings:
...
-            name: Virtual Rotater
-            description: Virtually rotates the shape, both clockwise and counter-clockwise.
+            name: Pemutar Virtual
+            description: Memutar bentuk secara virtual, searah jarum jam dan tidak searah jarum jam.
         unstacker:
-            name: Virtual Unstacker
-            description: Virtually extracts the topmost layer to the right output and the
-                remaining ones to the left.
+            name: Pemisah Tumpukan Virtual
+            description: Memisahkan lapisan teratas secara virtual ke output kanan dan
```
Comment (KaiSforza) [inline translations/base-ind.yaml:584]: Trailing space

File: translations/base-ind.yaml [hunk 6]
```diff
@@ -489,123 +489,121 @@ buildings:
...
-            name: Virtual Unstacker
-            description: Virtually extracts the topmost layer to the right output and the
-                remaining ones to the left.
+            name: Pemisah Tumpukan Virtual
+            description: Memisahkan lapisan teratas secara virtual ke output kanan dan 
+                sisanya ke output kiri.
         stacker:
-            name: Virtual Stacker
-            description: Virtually stacks the right shape onto the left.
+            name: Penumpuk Virtual
```
Comment (KaiSforza) [inline translations/base-ind.yaml:587]: Trailing space

File: translations/base-ind.yaml [hunk 7]
```diff
@@ -489,123 +489,121 @@ buildings:
...
-            name: Virtual Stacker
-            description: Virtually stacks the right shape onto the left.
+            name: Penumpuk Virtual 
+            description: Menumpuk bentuk kanan ke bentuk kiri secara virtual.
         painter:
-            name: Virtual Painter
-            description: Virtually paints the shape from the bottom input with the shape on
-                the right input.
+            name: Pencat Virtual
+            description: Mengecat bentuk dari input bawah dengan warna
```
Comment (KaiSforza) [inline translations/base-ind.yaml:591]: Trailing space

**Comment by tobspr** [comment]
See comments above (Huge thanks to @KaiSforza for pointing it out!)

File: translations/base-ind.yaml (hunk 1 above)
Comment (KaiSforza) [inline translations/base-ind.yaml:7]: Still has trailing space, everything else is fine.

Comment (KaiSforza) [reply translations/base-ind.yaml:7]: Note: I have no knowledge of these languages, i'm just critiquing the yaml syntax and stuff.

**Review by KaiSforza — APPROVED** [review]
yaml is good, only error left is in es.

**Comment by tobspr** [comment]
Thanks!

**Pull request merged**